packages/babel-jest/src/index.ts

```diff
@@ -269,11 +271,6 @@ export const createTransformer: CreateTransformer = userOptions => {
   };
 };
 
-const transformer: SyncTransformer<TransformOptions> = {
-  ...createTransformer(),
-  // Assigned here so only the exported transformer has `createTransformer`,
-  // instead of all created transformers by the function
-  createTransformer,
```

Comment:
One thing we need to check is that `{"\\.js$": ['babel-jest', {rootMode: "upward"}]}` still works, i.e. that we call. `createTransformer` on `babel-jest`.

`requireOrImportModule` will get `default` by... default, so `createTransformer` might be ignored for `babel-jest` if not present on the export

Reply:
At least now it is present on both the `default` _and_ a separate `export`, so it _should be fine_?